azure/init.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+# fail if trying to reference a variable that is not set.
+set -u
+# exit immediately if a command fails
+set -e
+# fail in a pipeline if any of the commands fails
+set -o pipefail
+
+# in redhat we need to enable default port for postgres
+firewall-cmd --add-port=5432/tcp
+
+# install pip since we will use it to install dependencies
+curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+python get-pip.py
+
+# install git to clone the repository
+yum install -y git
+
+# this is the username in our instances
+TARGET_USER=pguser
+
+# attach disk and mount it for data
+mkfs.ext4 -F /dev/sdc
+mkdir -p /home/${TARGET_USER}/citus-installation
+mount /dev/sdc /home/${TARGET_USER}/citus-installation
+rm -r /home/${TARGET_USER}/citus-installation/*
+chown ${TARGET_USER}:${TARGET_USER} /home/${TARGET_USER}/citus-installation
+
+# add the username to sudoers so that sudo command does not prompt password.
+# We do not want the password prompt, because we want to run tests without any user input
+echo '${TARGET_USER}     ALL=(ALL) NOPASSWD:ALL' >>/etc/sudoers
+
+su - ${TARGET_USER} <<'EOSU'
+ # add pg and local binaries to the path
+  echo "export PATH=${HOME}/pg-latest/bin/:${HOME}/.local/bin/:$PATH" >> ${HOME}/.bash_profile
+
+  cd ${HOME} && git clone --branch add-azure-support https://github.com/citusdata/test-automation.git
```

Comment:
Not forget to remove `--branch add-azure-support` :)

Reply:
Good point, we should do this right before merging